server/src/main/java/org/opensearch/action/admin/cluster/stats/ClusterStatsRequest.java

```diff
@@ -49,8 +50,13 @@ public class ClusterStatsRequest extends BaseNodesRequest<ClusterStatsRequest> {
 
     public ClusterStatsRequest(StreamInput in) throws IOException {
         super(in);
+        if (in.getVersion().onOrAfter(Version.V_3_0_0)) {
```

Comment:
We only send and recieve this on new versions, this is not sent in old versions as this may end up breaking during version upgrades.